### PR TITLE
feat(iac): github action should now use the new CI command

### PR DESCRIPTION
**Pull request by Paul-GitGuardian**
Depends on #628 to be deployed for the stable action to work.

Updates the IaC Github Action to use the new `ggshield iac scan ci` command instead of `ggshield iac scan all`.

### CODE REVIEW

**Comment by Paul-GitGuardian** [comment]
This changes the behavior of the action: it will now only block if new vulnerabilities are introduced.
Is it ok?

File: changelog.d/20230725_144801_paul.beslin.ext_update_github_action_iac_command.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Changed
+
+- The IaC Github Action now executes the new `ggshield iac scan ci` command. This means the action will fail only if the changes introduce a new vulnerability.
```
Comment (gg-teolohrer) [inline changelog.d/20230725_144801_paul.beslin.ext_update_github_action_iac_command.md:3]:
```suggestion
- The IaC Github Action now executes the new `ggshield iac scan ci` command. This means the action will fail only if the changes introduce a new vulnerability. To fail if any vulnerability is detected, use the `ggshield iac scan ci --all` command
```

**Review by gg-teolohrer — CHANGES_REQUESTED** [review]
The change in behavior seems OK to me, but we should describe the alternative in the changelog.

**Comment by cyrvan-b** [comment]
LGTM with the change on changelogs.

**Review by gg-teolohrer — APPROVED** [review]

**Review by cyrvan-b — APPROVED** [review]

**Comment by GG-HH** [comment]
@Paul-GitGuardian we should update the action files as well, the description will lack some options offered by the CI scan.

File: docker/actions-iac-entrypoint.sh [hunk 2]
```diff
@@ -12,4 +12,4 @@ git config --global --add safe.directory "$PWD"
 
 
 args=("$@")
-ggshield iac scan all -v ${args[@]}
+ggshield iac scan ci -v ${args[@]}
```
Comment (GG-HH) [inline docker/actions-iac-entrypoint.sh:15]:
```suggestion
ggshield iac scan ci ${args[@]}
```
We shouldn't have verbose enabled by default

**Pull request merged**